client/blocks/stats-navigation/index.js

```diff
@@ -277,15 +277,34 @@ class StatsNavigation extends Component {
 			!! ( statsAdminVersion && version_compare( statsAdminVersion, '0.9.0-alpha', '>=' ) );
 
 		const shouldRenderModuleToggler =
-			! isLegacy &&
 			isModuleSettingsSupported &&
 			AVAILABLE_PAGE_MODULES[ this.props.selectedItem ] &&
 			! hideModuleSettings &&
 			! gatedTrafficPage;
 
 		// @TODO: Add loading status of modules settings to avoid toggling modules before they are loaded.
 
-		const validNavItems = Object.keys( navItems ).filter( this.isValidItem );
+		/** @type {(keyof typeof allNavItems)[]} */
+		const navKeys = Object.keys( allNavItems );
+		const navItems = navKeys.filter( this.isValidItem ).map( ( key ) => {
+			const navItem = allNavItems[ key ];
+
+			if ( ! navItem ) {
+				throw new Error( `navItem is null for key: ${ key }` );
+			}
+
+			const intervalPath = navItem.showIntervals ? `/${ interval || 'day' }` : '';
+			const itemPath = `${ navItem.path }${ intervalPath }${ slugPath }`;
+			return {
+				name: key,
+				adminUrl: `${ adminUrl }admin.php?page=wc-admin&path=%2Fanalytics%2Foverview`,
```

Comment:
```suggestion
				storeAdminUrl: `${ adminUrl }admin.php?page=wc-admin&path=%2Fanalytics%2Foverview`,
```
To avoid getting confused with the original `adminUrl`.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/103350/commits/6a8a590c6cfc28055b53143451935c2aae3784d3.